akka-docs/src/main/paradox/stream/operators/Source/actorRef.md

```diff
@@ -16,11 +16,18 @@ Materialize an `ActorRef`, sending messages to it will emit them on the stream.
 a buffer but since communication is one way, there is no back pressure. Handling overflow is done by either dropping
 elements or failing the stream; the strategy is chosen by the user.
 
+The stream can be completed successfully by sending the actor reference a `akka.actor.Status.Success`.
+If the content is `akka.stream.CompletionStrategy.immediately` the completion will be signaled immediately.
+Otherwise, if the content is `akka.stream.CompletionStrategy.draining` (or anything else)
+already buffered elements will be sent out before signaling completion.
+Sending `akka.actor.PoisonPill` will signal completion immediately but this behavior is deprecated and scheduled to be removed.
+Using `akka.actor.ActorSystem.stop` to stop the actor and complete the stream is *not supported*.
+
```

Comment:
Do the above two points warrant a `@@@note`? Could consider it, because one is a pending deprecation, the other is an unsupported.